home/home4_1/Director.java

```diff
@@ -0,0 +1,17 @@
+package home4_1;
+public class Director {
+    private PizzaBuilder builder;
+
+    public void setBuilder(PizzaBuilder builder) {
+        this.builder = builder;
+    }
+
+    public void constructPizza(String first, String second, String third, String fourth,
```

Comment:
maybe using vector of pairs will be more relevant way to pass such a list of arguments